source/text/vss-characters.ads

```diff
@@ -97,4 +97,13 @@ package VSS.Characters is
      (Self : Virtual_Character) return General_Category;
    --  Return General_Category property for given character.
 
+   function Get_Lowercase (Self : Virtual_Character) return Boolean;
```

Comment:
Why not `Is_Lowercase`, `Is_Uppercase`, `Is_Cased`?

Reply:
Is_Lowercase is reserved for operation on String type. Get_Lowercase returns property of the single character, and String.Is_Lowercase of the string of single character may be different from Character.Get_Lowercase of the same character. Different names highlights that it equality is not expected.